AUTHORS.rst

```diff
@@ -1,6 +1,17 @@
 
-=====================================
-AUTHORS
-=====================================
+=======
+Authors
+=======
 
-...
+sphinx_typo3_theme is written and maintained by Martin Bless
+<martin.bless@mbless.de>.
+
+Major developer is Benjamin Kott <benjamin.kott@typo3.org> who created the
```

Comment:
please use `benjamin.kott@outlook.com` i do not have a .org mail address.

Reply:
Thank you for reviewing. I'll change the email.